### PR TITLE
WF-64352-Add KB links in Find and Replace UG

### PR DESCRIPTION
**Pull request by elizabethatienoo**
<p>Kindly review the below merge request and let me know your feedback.</p>
<table>
<tbody>
<tr>
<td>
<p>Task Link</p>
</td>
<td>
<p><a href="https://syncfusion.atlassian.net/browse/EJDOTNETCORE-4001">https://syncfusion.atlassian.net/browse/EJDOTNETCORE-4001</a></p>
<p><a href="https://syncfusion.atlassian.net/browse/WF-64148">https://syncfusion.atlassian.net/browse/WF-64148</a>
<p><a href="https://syncfusion.atlassian.net/browse/EJ2CORE-874">https://syncfusion.atlassian.net/browse/EJ2CORE-874</a></p>
</td>
</tr>
<tr>
<td>
<p>Description</p>
</td>
<td>
<p>How to find and replace text inside table?</p>
<p>How to find and replace text in headers and footers of Word document?</p>
<p>How to find and replace placeholder with page break in Word document?</p>
</td>
</tr>
<tr>
<td>
<p>Solution</p>
</td>
<td>
<p>Added FAQ in UG pages to redirect for KB</p>
</td>
</tr>
</tbody>
</table>

### CODE REVIEW

File: File-Formats/DocIO/Working-with-Find-and-Replace.md [hunk 1]
```diff
@@ -2585,4 +2585,6 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 * [How to replace the particular text with hyperlink in Word document](https://www.syncfusion.com/kb/11774/how-to-replace-the-particular-text-with-hyperlink-in-word-document)
 * [How to find the empty paragraphs in the Word document using Essential DocIO](https://www.syncfusion.com/kb/3956/how-to-find-the-empty-paragraphs-in-the-word-document-using-essential-docio)
-* [How to replace text in a Word document with HTML](https://www.syncfusion.com/kb/13627/how-to-replace-text-in-a-word-document-with-html)
\ No newline at end of file
+* [How to replace text in a Word document with HTML](https://www.syncfusion.com/kb/13627/how-to-replace-text-in-a-word-document-with-html)
+* [How to find and replace text inside table in Word document](https://www.syncfusion.com/kb/13640/how-to-find-and-replace-text-inside-table-in-word-document)
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Find-and-Replace.md:2589]: Can you please use this link https://www.syncfusion.com/kb/13640/how-to-find-and-replace-text-inside-table-in-word-document

File: File-Formats/XlsIO/Create-read-edit-excel-files-in-asp-net-core-c-sharp.md [hunk 2]
```diff
@@ -462,10 +462,8 @@ IApplication application = excelEngine.Excel;
 //Assigns default application version
 application.DefaultVersion = ExcelVersion.Xlsx;
 
-//A existing workbook is opened.              
-string basePath = _hostingEnvironment.WebRootPath + @"\XlsIO\Sample.xlsx";              
-FileStream sampleFile = new FileStream(basePath, FileMode.Open);
-
+//A existing workbook is opened.
```
Comment (MohanaselvamJothi) [inline File-Formats/XlsIO/Create-read-edit-excel-files-in-asp-net-core-c-sharp.md:465]: Can you please give PR to hotfix branch.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Find-and-Replace.md (hunk 1 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Find-and-Replace.md:2589]: This was the same link used before ,do you mean https://www.syncfusion.com/kb/13641/how-to-find-and-replace-text-inside-table-in-word-document which is for EJ2?

**Review by MohanaselvamJothi — APPROVED** [review]

**Comment by MohanaselvamJothi** [comment]
rebuild

**Comment by MohanaselvamJothi** [comment]
rebuild

**Comment by MohanaselvamJothi** [comment]
rebuild

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**